rocky/reports/report_types/ipv6_report/report.py

```diff
@@ -48,6 +48,8 @@ def generate_data(self, input_ooi: str, valid_time: datetime) -> Dict[str, Any]:
 
         results = {}
         for hostname in hostnames:
+            if ooi.reference.class_type == IPAddressV6:
+                return {hostname.name: {"enabled": True} for hostname in hostnames}
```

Comment:
If there's only one pair in the dictionary, you might as well just assign a boolean (enabled value) instead. Booleans are less complex and use less resources; and within this report context it's clear what the boolean means.

Reply:
I agree, we are going to revisit the structures of the sub-reports in another PR. This extra line is just there to avoid unnecessary queries without changing the structure of the outcome.

Reply:
Okay, just make sure not to forget this one :)